### DISCUSSION TITLE
Dashboard - Home discussion

### DISCUSSION THREAD

**Discussion by SaxonF** [supabase/supabase#32358] [General]
Hey everyone,

If we were to look at refreshing the `Home` page of the Supabase Dashboard, what would you hope to see? Have a think about what actions you perform regularly, or data or metrics that needs more visibility, or even challenges you've had in the past during onboarding. Consider your workflows around logging/monitoring and database health in general. What can we do to save more of your time? What shortcuts can we provide? What metrics are important to you? What do you use the current home screen for primarily? 

This is an open discussion so all ideas and feedback welcome and appreciated!

**Comment by Anirudh181013** [comment]
Hey there Saxon, I've noticed few things:-
on flaws side:-
1. icon-only buttons dont show meaning when hovered on
2. menu headings(Projects, Organizations, Accounts) look identical to their list items, can be made to look visually separate
3. the lines that separate different sections on the dashboard look a little out of sync with the page

Functionality-wise the site is very well done, thanks a lot for such an application through the web. (PS: eventhough i'm criticising this much, I am not experienced enough to tell all this, but just trying to contribute if my feedback is valuable)
